fbg-server/src/proto/user_pb.d.ts

```diff
@@ -0,0 +1,37 @@
+import * as jspb from 'google-protobuf';
+
+export class NewUserReq extends jspb.Message {
```

Comment:
this seems auto-generated? do we need to commit it? cant it be part of our build steps?

Reply:
if we need to commit it, just make sure we have a folder that makes it clear it is autogen